source/containers/FormField/FormField.js

```diff
@@ -212,6 +219,8 @@ const FormField = ({
   if (input && input.focusEvent) inputCompProps[input.focusEvent] = onInputFocus;
   if (input && input.helpInComponent) inputCompProps[input.helpProp || 'help'] = help;
 
+  if (input && inputType === 'repeaterField' && input.onAddAnswerEvent) inputCompProps[input.onAddAnswerEvent] = onInputAddAnswer;
```

Comment:
a little less code perhaps:

```js
if(inputType === 'repeaterField' && !!input?.onAddAnswerEvent) inputCompProps[input.onAddAnswerEvent] = onInputAddAnswer;
```

Reply:
I do not like double nots in javascript. It's a very code-golfy javascript-y way of forcing a boolean, although I can see the appeal here.

Looking into this code it's a bit it's very messy. If I'm not mistaken it exits out early on line 178 if `input === undefined` meaning that for the rest of the code there should be no reason to check `if(input && ..` at all (unless it's somehow `null` instead of `undefined`.

For now I would just replace this line with `if(input.onAddAnswerEvent)` since we probably don't need to check for a repeater field anyway; if `onAddAnswerEvent` is specified then we want it regardless of input type right?